README.md

```diff
@@ -1,90 +1,166 @@
 
-## Joining the discussion
+The individual libraries are developed to be agnostic about the utilized [Distributed Ledger Technology (DLT)](https://en.wikipedia.org/wiki/Distributed_ledger), with the exception of the [IOTA](https://www.iota.org) integration and higher level libraries. Written in stable Rust, it has strong guarantees of memory safety and process integrity while maintaining exceptional performance.
 
-If you want to get involved in discussions about this framework, or you're looking for support, go to the #identity-discussion channel on [Discord](http://discord.iota.org).
+> :warning: **WARNING** :warning:
+> 
+> This library is currently in its **alpha stage** and **under development** and might undergo large changes!
+> Until a formal third-party security audit has taken place, the [IOTA Foundation](https://www.iota.org/) makes no guarantees to the fitness of this library. As such, it is to be seen as **experimental** and not ready for real-world applications.
+> Nevertheless, we are very interested in feedback about user experience, design and implementation, and encourage you to reach out with any concerns or suggestions you may have.
+
+## Documentation and Resources
+
+- [API Reference](https://identity-docs.iota.org/docs/identity/index.html): Package documentation (cargo docs).
+- [Identity Documentation Pages](https://identity-docs.iota.org/welcome.html): Supplementing documentation with context around identity and simple examples on library usage.
+- [Examples in /examples folder](https://github.com/iotaledger/identity.rs/tree/main/examples): Practical code snippets to get you started with the library.
+- [IOTA Identity Experience Team Website](https://iota-community.github.io/X-Team_IOTA_Identity/): Website for a collaborative effort to provide help, guidance and spotlight to the IOTA Identity Community through offering feedback and introducing consistent workflows around IOTA Identity.
+
+## Getting Started
+
+If you want to include IOTA Identity in your project, simply add it as a dependency in your `cargo.toml`:
+```rust
+[dependencies]
+identity = { git = "https://github.com/iotaledger/identity.rs", branch = "main"}
+```
+
+To try out the [examples](https://github.com/iotaledger/identity.rs/tree/main/examples), you can also do this:
+
+1. Clone the repository, e.g. through `git clone https://github.com/iotaledger/identity.rs `
+2. Build the repository with `cargo build `
```

Comment:
do you need this step?

Reply:
technically you don't need step 2 because step 3 will compile it anyways, but you definitely need 1. i left it in for completion's sake